src/components/BookCard.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+export default function BookCard({ id, cover, title, author }) {
+  return (
+    <Card key={id}>
+      <CoverWrapper>
+        <img src={cover} alt="" width="75" height="110" />
+      </CoverWrapper>
+      <section>
+        <h2>{title}</h2>
+        <Author>{author}</Author>
+      </section>
+    </Card>
+  )
+}
+
+BookCard.propTypes = {
+  id: PropTypes.string,
+  cover: PropTypes.any,
+  title: PropTypes.string,
+  author: PropTypes.string,
+}
+
+const Card = styled.section`
+  background-color: #f3f3f3;
+  display: grid;
+  grid-template-columns: 2fr 5fr;
+`
+const CoverWrapper = styled.span`
+  display: grid;
+  align-content: center;
+  justify-content: center;
+  padding: 10px;
+`
+const Author = styled.h3`
```

Comment:
Why is it an `h3` element instead of `h1` or `h2`?

Reply:
Thanks, just changed it to an h2 as it is a heading for one element, but not for the entire page